module-6.md

```diff
@@ -16,11 +16,25 @@ Planned:      1.25 hour
 
 ## 2: Hands-on and material exploration
 
-- Build a 900 MHz LoRa sensor network with WiFi mesh network as backhaul
-- Use case for a community network that functions without internet connectivity
+### Build a 900 MHz LoRa sensor network with WiFi mesh network as backhaul
 
-## 3: Discussion and reflection activity
+- Motivation to use low power networks:  
+  * Energy, cost, simplicity  
+  * Examples of low power networks by communities (eg. citizen science projects, resilient communication)
+
+- Sending Information Wirelessly
+  * Quick recap of Module 3 (Frequency, Amplitude, and Phase from Metamesh)
+  * Modulation techniques - FM, AM, Chirp
+  * How does LoRa work? What are some alternatives?
+  * ISM band and spectrum allocation in Canada
+
+- Low Power Microcontrolers
```

Comment:
@benhylau it could be possible to do a hands-on demo of Pi-to-Pi communication over LoRa instead of using Arduino's. That may be easier for people because they would already be familiar with Pi's at this point - however it won't be low-power anymore. Thoughts?

Reply:
Just be careful logic levels are different between Arduino and the pi... Depending on what you do you may need a logic level. Converter..

I don't think not using low level micro processors is an issue in an educational setting.. It's more to intrude a concept to lora. We can note that there are low powered processors but I think the learning curve would be maybe to high to throw Arduino into all this to.